libnymea-core/integrations/thingmanagerimplementation.cpp

```diff
@@ -890,14 +892,14 @@ BrowseResult *ThingManagerImplementation::browseThing(const ThingId &thingId, co
     BrowseResult *result = new BrowseResult(thing, this, itemId, locale, this, 30000);
 
     if (!thing) {
-        qCWarning(dcThingManager()) << "Cannot browse thing. No such thing:" << thingId.toString();
+        qCWarning(dcThingManager()) << "Cannot browse thing. No such thing with ID:" << thingId.toString();
```

Comment:
either "no such thing" or "no thing with ID..."

Reply:
done also in the other places